src/typescript/12-generics/generics.md

```diff
@@ -370,61 +310,21 @@ root.right.value      //-> "Tom"
 root.right.left.value //-> "Matthew"
 ```
 
-### Verify Your Solution
-
-
-✏️ Run the following to verify your solution:
-
-```shell
-npm run 6-generics
-```
-
+<strong>Have issues with your local setup?</strong> You can use either [StackBlitz](https://stackblitz.com/fork/github/bitovi/academy/tree/main/exercises/typescript/12-generics/02-problem?file=src/tree-node.ts) or [CodeSandbox](https://codesandbox.io/p/devbox/github/bitovi/academy/tree/main/exercises/typescript/12-generics/02-problem?file=src/tree-node.ts) to do this exercise in an online code editor.
 
-### The solution
+### Solution 2
 
 <details>
 <summary>Click to see the solution</summary>
 
-✏️ Update `6-tree-node.ts` to the following:
+Update `tree-node.ts` to the following:
```

Comment:
This should use the pencil icon to be consistent with everything else.

Also, the indentation in the solution file is wrong.